modules/post/multi/gather/jenkins_gather.rb

```diff
@@ -231,9 +231,9 @@ def pretty_print_gathered
     @ssh_keys.uniq.each do |e|
       print_good("SSH Key")
       print_status(" ID: #{e[0]}")
-      print_status(" Description: #{e[1]}") if !e[1].nil? || !e[1].empty?
-      print_status(" Passphrase:  #{e[2]}") if !e[2].nil? || !e[2].empty?
-      print_status(" Username:    #{e[3]}") if !e[3].nil? || !e[3].empty?
+      print_status(" Description: #{e[1]}") if !e[1].nil? || !e[1].blank?
+      print_status(" Passphrase:  #{e[2]}") if !e[2].nil? || !e[2].blank?
+      print_status(" Username:    #{e[3]}") if !e[3].nil? || !e[3].blank?
```

Comment:
`.blank?` would also check if the object is `nil` as well and so the first part of this code would not be needed. Might I suggest this instead to simplify the code?

```suggestion
      print_status(" Description: #{e[1]}") if !e[1].blank?
      print_status(" Passphrase:  #{e[2]}") if !e[2].blank?
      print_status(" Username:    #{e[3]}") if !e[3].blank?
```

For reference this is what happens with a `.blank?` call on `nil` object:

```
>> nil.blank?
=> true
>> !nil.blank?
=> false
>> 
```

https://blog.appsignal.com/2018/09/11/differences-between-nil-empty-blank-and-present.html has more details on this if you wish to read up more about this 👍

Reply:
Fixed with latest updates.